src/xunit.performance.api/ProcessExitInfo.cs

```diff
@@ -11,18 +12,31 @@ public sealed class ProcessExitInfo
         /// <summary>
         /// Initializes a new instance of a ProcessExitInfo class with the run process.
         /// </summary>
-        /// <param name="process"></param>
-        public ProcessExitInfo(Process process)
+        /// <param name="process">The terminated process.</param>
+        /// <param name="startTime">The time that the associated process was started.</param>
+        /// <param name="exitTime">The time that the associated process exited.</param>
+        internal ProcessExitInfo(Process process, DateTime startTime, DateTime exitTime)
         {
             if (process == null)
                 throw new ArgumentNullException(nameof(process));
             if (!process.HasExited)
                 throw new InvalidOperationException($"{process.ProcessName} has not exited.");
+            if (exitTime < startTime)
+                throw new InvalidOperationException($"Process.ExitTime: {exitTime}, is less than the Process.StartTime: {startTime}.");
 
             ProcessId = process.Id;
             ExitCode = process.ExitCode;
-            StartTime = process.StartTime;
-            ExitTime = process.ExitTime;
+
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```

Comment:
The change here seems reasonable to me, but I think rather than special casing Windows, just do the same thing on all platforms.  Otherwise, our methodology is different between operating systems, and that makes both the harness and data interpretation more complicated.

Reply:
Thanks, makes sense.